apps/console/src/app.tsx

```diff
@@ -18,13 +18,14 @@
 
 import { DecodedIDTokenPayload, useAuthContext } from "@asgardeo/auth-react";
```

Comment:
There are lint warnings. Can we get them fixed?

Reply:
Done